README.md

```diff
@@ -63,20 +61,38 @@ Default `docker-compose.yml` file runs Temporal server with Cassandra. There are
 | docker-compose-postgres.yml  | PostgreSQL |
 | docker-compose-cockroach.yml | CockroachDB |
 
-Use one of the these files to run corresponding Temporal configuration. For example to run Temporal with MySQL
-and Elasticsearch for enhance visibility queries run:
+### Use a custom image configuration
+
+If you want, you can even use a custom Docker image of the Temporal Server.
+
+Clone the main Temporal Server repo: [https://github.com/temporalio/temporal](https://github.com/temporalio/temporal):
 
 ```bash
-$ docker-compose -f docker-compose-mysql-es.yml up
+git clone https://github.com/temporalio/temporal.git
+```
+
+In the following command, replace **<YOUR_TAG>** and **<YOUR_COMMIT>** to build the custom Docker image:
+
+```bash
+git checkout <YOUR_COMMIT>
+docker build . -t temporalio/auto-setup:<YOUR_TAG> --build-arg TARGET=auto-setup
+```
+
+Next, in the `docker-compose.yml` file, replace the `services.temporal.image` configuration value with **<YOUR_TAG>**.
+
+Then run the `docker-compose up` command:
+
+```bash
+docker-compose up
 ```
 
 ## Quickstart for production
 
-In a typical production setting, dependencies (such as `cassandra` or `elasticsearch`) are managed/started independently of the Temporal server.
+In a typical production setting, dependencies such as `cassandra` or `elasticsearch` are managed/started independently of the Temporal server.
 To use the container in a production setting, use the following command:
 
-```plain
-$ docker run -e CASSANDRA_SEEDS=10.x.x.x                -- csv of cassandra server ipaddrs
+```bash
+docker run -e CASSANDRA_SEEDS=10.x.x.x                -- csv of cassandra server ipaddrs
```

Comment:
$ docker run -e CASSANDRA_SEEDS=10.x.x.x                -- csv of cassandra server ipaddrs